### PR TITLE
Add CORS support

### PR DESCRIPTION
**Pull request by makenosound**
Using `rack-cors`. @dylanwolff / @dNitza Would be good to get you to sanity check this!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ gem "rake"
 gem "puma"
 gem "dry-web", "~> 0.2"
 gem "dry-web-roda", "~> 0.1"
+gem "rack-cors", :require => "rack/cors"
```
Comment (timriley) [inline Gemfile]: No need for the `require` here, since in these apps we never do `Bundler.require` in these apps :)

**Comment by dNitza** [comment]
Looks good to me! 👍

File: Gemfile (hunk 1 above)
Comment (makenosound) [inline Gemfile]: Ah, ta — I did not really understand that and now I do. Removed!

**Comment by makenosound** [comment]
Suhweet. Mergin’

**Pull request merged**